Issue-Embeddings/flask_app/inference.py

```diff
@@ -148,31 +159,50 @@ def df_to_emb(self, dataframe:pd.DataFrame, bs=150) -> np.ndarray:
         (200, 2400)
         """
         new_df = self.process_df(dataframe)
-        text_list = new_df['text'].to_list()
-        
-        numercalized_docs = []
+        #text_list = new_df['text'].to_list()
```

Comment:
nit: can we delete this commented out code?

Reply:
👍